src/service/get-service.js

```diff
@@ -96,6 +101,60 @@ class GetService extends OperationService {
                     tokenId,
                     keyword,
                 );
+
+                if (paranetSync) {
```

Comment:
Generally not sure if this is the right place for sync logic.

I think it shouldn't be the responsibility of getting service to do asset sync logic, I would suggest moving this logic to paranet-sync-command and just getting the result of the network get from there, and continue inserting and moving data.

Reply:
IIRC Djordje said it's OK, as it's already syncing assets above?